words.txt

```diff
@@ -0,0 +1,400 @@
+find
+any
+new
+work
+part
+take
+get
+place
+made
+live
+where
+after
+back
+little
+only
+round
+man
+year
+came
+show
+every
+good
+me
```

Comment:
Can we set the min length of words to be 3 char?

Reply:
On another note, should we have a max length too?

Reply:
Sure, I think a min and max length is good. min 3, max 8?? suggestions?

Reply:
words <= 8 sound fine to me 👍